doc/boostdoc.css

```diff
@@ -0,0 +1,92 @@
+/*! Asciidoctor default stylesheet | MIT License | https://asciidoctor.org */
```

Comment:
Why does this file say "Asciidoctor default stylesheet | MIT license"? Is this even correct? Where is this file used?

Reply:
Actually this file is not used, it was from earlier development before we moved things to the boostlook repo.